crime-evidence/build.gradle

```diff
@@ -91,6 +91,9 @@ dependencies {
     implementation group: 'com.amazonaws', name: 'aws-xray-recorder-sdk-slf4j', version: versions.xRayVersion
     implementation group: 'com.amazonaws', name: 'aws-xray-recorder-sdk-spring', version: versions.xRayVersion
 
+    implementation group: 'io.sentry', name: 'sentry-spring-boot-starter'
```

Comment:
We already added the sentry dependencies here:

dependencyManagement {
    dependencies {
        dependencySet(group: 'io.sentry', version: versions.sentryVersion) {
            entry 'sentry-spring-boot-starter'
            entry 'sentry-logback'
        }
    }
}